packages/block-editor/src/components/iframe/index.js

```diff
@@ -336,13 +374,16 @@ function Iframe( {
 			`${ containerWidth }px`
 		);
 		iframeDocument.documentElement.style.setProperty(
-			'--wp-block-editor-iframe-zoom-out-prev-container-width',
-			`${ prevContainerWidthRef.current }px`
+			'--wp-block-editor-iframe-zoom-out-outer-container-width',
```

Comment:
I would maybe rename this to `--wp-block-editor-iframe-zoom-out-scale-container-width` instead since this is the width of the `block-editor-iframe__scale-container`.

Reply:
We may not always call it scale container, so I'd be fine leaving this one named as is.

Reply:
Right, it may be changed, but for the current reader, the connection between the two would be more clear if the variable was named the same as the container that it's setting the width for.